NH-46957: add notice (#38)
solarwindscloud/solarwinds-apm-ruby: README.md

# Notice

> As of 2023-07-21, we regret to announce that **solarwinds-apm-ruby-gen-1** is not in active development. We have shift our focus on opentelemetry-based apm library. The new repository is located in https://github.com/solarwindscloud/solarwinds-apm-ruby

Reviewer: ```suggestion
This is the renamed repository for `solarwinds-apm` versions below 6, which is now deprecated in favour of the OpenTelemetry-based `solarwinds-apm` starting from version 6, located at https://github.com/solarwindscloud/solarwinds-apm-ruby.
```

> Please be aware of the following implications:

Reviewer: ```suggestion
```

> 1. **Limited Updates**: There will be limited new feature updates, bug fixes, or enhancements to the **solarwinds-apm-ruby-gen-1** from our official development team.

Reviewer: ```suggestion
```

> 2. **Limited Maintenance**: We will still maintain the repository or address any issues or pull requests submitted, but the duration may take longer.

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
Please note this codebase is no longer in active development as we focus on enhancing `solarwinds_apm` version 6+.  Version 5 releases will be limited to critical updates and fixes, and issues or pull requests submitted to this repository may have a longer response time.
```
